FirebaseCore/Sources/Private/FIRHeartbeatLogger.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 Google
```

Comment:
Since private headers are going away in Firebase 9, it might make sense to start the FirebaseCoreInternal library now just with the heartbeat implementation now and add the rest of the privates when we do the breaking change.  See #8973